src/uPlot.js

```diff
@@ -381,8 +381,8 @@ export default function uPlot(opts, data, then) {
 
 	function delSeries(i) {
 		series.splice(i, 1);
-		FEAT_LEGEND && legendRows.splice(i, 1)[0][0].parentNode.remove();
-		FEAT_CURSOR && cursorPts.splice(i, 1)[0].remove();
+		FEAT_LEGEND && legendRows[i] && legendRows.splice(i, 1)[0][0].parentNode.remove();
```

Comment:
This could also be 
```js
		FEAT_LEGEND && showLegend && legendRows.splice(i, 1)[0][0].parentNode.remove();
```

but I'm not sure cursorPts has a similar bool to check so I went with this so that they are consistent.

Reply:
yeah, i was gonna ask this.

> `FEAT_LEGEND && showLegend`

i like this more, since we already have the dedicated flag and it's used evrywhere.

> but I'm not sure cursorPts has a similar bool to check so I went with this so that they are consistent.

updateCursor uses `FEAT_CURSOR && cursorPts.length > 1`, so let's do the same here.

Reply:
:+1: